src/components/calculator.js

```diff
@@ -10,66 +23,70 @@ export default function theCalculator() {
       </header>
       <main>
         <section>
-          <div id="Myscreen">
-            <input id="screen" value="0" readOnly />
+          <div id="screen">
+            {state.total}
+            {' '}
+            {state.operation}
+            {' '}
+            {state.next}
           </div>
           <div id="keyboard">
             <div className="btnBox">
-              <button id="btnAC" type="button"> AC </button>
+              <button id="btnAC" type="button" onClick={eventCalculation} value="AC">AC</button>
             </div>
             <div className="btnBox">
-              <button id="btn+/-" type="button"> +/- </button>
+              <button id="btn+/-" type="button" onClick={eventCalculation} value="+/-"> +/- </button>
             </div>
             <div className="btnBox">
-              <button id="btnModulo" type="button"> % </button>
+              <button id="btnModulo" type="button" onClick={eventCalculation} value="%"> % </button>
             </div>
             <div className="btnBox orj">
-              <button id="btnDivision" type="button"> ÷ </button>
+              <button id="btnDivision" type="button" onClick={eventCalculation} value=" ÷"> ÷ </button>
```

Comment:
- [ ] when i click this operation I get an error, this is because of the space in the value string,  kindly change this  `value=" ÷"` to `value="÷"`
![Screenshot from 2023-04-27 18-51-17](https://user-images.githubusercontent.com/66097371/234951659-6c624aa0-1afd-47dd-8697-f9a895625f3f.png)